### DISCUSSION TITLE
PDF encrypted

### DISCUSSION THREAD

**Discussion by Name-None** [sumatrapdfreader/sumatrapdf#3930] [Q&A] [answered]
I was looking at the PDF and suddenly it showed that the PDF file was encrypted. Do you know why? I only opened Sumatra one software

**Comment by GitHubRulesOK** [comment]
SumatraPDF will only RE-use encryption if it was already an encrypted file, (If a password is needed to open PDF,then a password is needed to save amended PDF)
It cannot add new encryption nor add encoding like Acrobat Reader (using standard editor functions) may do when asking to save.

**Reply by Name-None** [reply]
Yeah, I know Sumatra doesn't have encryption function, but I did suddenly turn the file into an encrypted file without opening any other software. I know it sounds strange, haha

**Reply by GitHubRulesOK** [reply]
IF the file was faulty due to some file unlocking application not doing its job correctly, then it is possible MuPDF core engine will have attempted to fix any faults (Same as Acrobat does) then on closure, the new corrected behaviour may be, to confirm the password that is already encrypted inside the PDF, but SumatraPDF should never do that on its own , it does not have that ability.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/6070015a-6597-48b4-9aa9-b46e441f991a)
The problem of repeated ask for password can arise when adding annotation if the filename or content is changing.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/1fabddea-86b4-453a-b476-2b84196af6ae)

**Reply by Name-None** [reply, marked as answer]
I just realized that this PDF file is actually restricted from adding comments. But when using sumatra, there was no prompt that the file does not allow comments, and instead, comments can still be made. It's amazing, haha. After I lifted the restriction on this file, there was no problem with it anymore.

Sorry to bother you. Thank you very much
